tests/longhaul/block_on_k8s_test.go

```diff
@@ -44,7 +49,7 @@ func (s *K8sBlockLongHaulSuite) SetupSuite() {
 	assert.Nil(s.T(), err)
 
 	s.installer = installer.NewK8sRookhelper(s.kh.Clientset)
-	if !s.kh.IsRookInstalled() {
+	if !s.kh.IsRookInstalled("rook") {
```

Comment:
do we not have a const for the default namespace? (and better not to use "rook" in the tests so we make sure we work in different namespaces from dev env)

Reply:
that was in a different package. Add them here too